.env.sample

```diff
@@ -26,4 +26,5 @@ KAFKA_URL = "172.31.0.4:9092"
 KAFKA_GROUP_ID = "projects"                                                     // Kafka group id                                
 
 # SUBMISSION TOPIC
-SUBMISSION_TOPIC = "dev.sl.projects.submissions"                                // Kafka topic name for pushing projects submissions
\ No newline at end of file
+SUBMISSION_TOPIC = "dev.sl.projects.submissions"                                // Kafka topic name for pushing projects submissions
+IMPROVEMENT_PROJECT_SUBMISSION_TOPIC = "dev.sl.projects.imp_submissions"        // project submission topic 
```

Comment:
@rakeshSgr - Make it "dev.sl.projects.submissions"

Reply:
@rakeshSgr the key can be "PROJECT_SUBMISSION_TOPIC"

Reply:
resolved